packages/drawer/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## 1.0.0 (2021-08-30)
+
+### Features
+
+- **drawer:** implement drawer component LS-3567
```

Comment:
You should not add this file manually. It should be auto generated by Lerna

Reply:
Removed in ed4a63db31e0eb16cd9e69adf3e0cb4c5205aecd